docs/configuring-dns.md

```diff
@@ -28,18 +28,22 @@ If you are using Cloudflare DNS, make sure to disable the proxy and set all reco
 
 ## DNS settings for optional services/features
 
-| Type  | Host                         | Priority | Weight | Port | Target                 |
-| ----- | ---------------------------- | -------- | ------ | ---- | ---------------------- |
-| SRV   | `_matrix-identity._tcp`      | 10       | 0      | 443  | `matrix.<your-domain>` |
-| CNAME | `dimension`                  | -        | -      | -    | `matrix.<your-domain>` |
-| CNAME | `jitsi`                      | -        | -      | -    | `matrix.<your-domain>` |
-| CNAME | `stats`                      | -        | -      | -    | `matrix.<your-domain>` |
-| CNAME | `goneb`                      | -        | -      | -    | `matrix.<your-domain>` |
-| CNAME | `sygnal`                     | -        | -      | -    | `matrix.<your-domain>` |
-| CNAME | `ntfy`                       | -        | -      | -    | `matrix.<your-domain>` |
-| CNAME | `hydrogen`                   | -        | -      | -    | `matrix.<your-domain>` |
-| CNAME | `cinny`                      | -        | -      | -    | `matrix.<your-domain>` |
-| CNAME | `buscarron`                  | -        | -      | -    | `matrix.<your-domain>` |
+| Type  | Host                           | Priority | Weight | Port | Target                      |
+| ----- | ------------------------------ | -------- | ------ | ---- | --------------------------- |
+| SRV   | `_matrix-identity._tcp`        | 10       | 0      | 443  | `matrix.<your-domain>`      |
+| CNAME | `dimension`                    | -        | -      | -    | `matrix.<your-domain>`      |
+| CNAME | `jitsi`                        | -        | -      | -    | `matrix.<your-domain>`      |
+| CNAME | `stats`                        | -        | -      | -    | `matrix.<your-domain>`      |
+| CNAME | `goneb`                        | -        | -      | -    | `matrix.<your-domain>`      |
+| CNAME | `sygnal`                       | -        | -      | -    | `matrix.<your-domain>`      |
+| CNAME | `ntfy`                         | -        | -      | -    | `matrix.<your-domain>`      |
+| CNAME | `hydrogen`                     | -        | -      | -    | `matrix.<your-domain>`      |
+| CNAME | `cinny`                        | -        | -      | -    | `matrix.<your-domain>`      |
+| CNAME | `buscarron`                    | -        | -      | -    | `matrix.<your-domain>`      |
+| MX    | `matrix`                       | 10       | 0      | -    | `matrix.<your-domain>`      |
+| TXT   | `matrix`                       | -        | -      | -    | `v=spf1 ip4:<your-ip> -all` |
+| TXT   | `_dmarc.matrix`                | -        | -      | -    | `v=DMARC1; p=quarantine;`   |
+| TXT   | `postmoogle._domainkey.matrix` | -        | -      | -    | get it from `!pm dkim`      |
```

Comment:
I wonder if we should add some `Matrix Component` column at the end, which specifies which component requires this service.

On each row, we can link to specific component's `docs/` page.

Reply:
Tried to add it, looks **very** bad

Reply:
Let's leave it as it is then